CHANGES-5.0.md

```diff
@@ -8,6 +8,7 @@
 
 * Check ACLs for last will testament topic before publishing the message. [#8930](https://github.com/emqx/emqx/pull/8930)
 * Fix GET /listeners API crash When some nodes still in initial configuration. [#9002](https://github.com/emqx/emqx/pull/9002)
+* Fix empty variable interpolation in authentication and authorization [#8963](https://github.com/emqx/emqx/pull/8963)
```

Comment:
Please specify how it is fixed exactly, since it may have implications for the users.

Reply:
✅